### PR TITLE
[4.x] Fix bad merge

### PR DESCRIPTION
**Pull request by yschimke**

### CODE REVIEW

**Comment by yschimke** [comment]
@swankjesse test only, requesting post review.

File: okhttp-testing-support/src/main/kotlin/okhttp3/internal/io/InMemoryFileSystem.kt [hunk 1]
```diff
@@ -119,4 +119,7 @@ class InMemoryFileSystem : FileSystem, TestRule {
   }
 
   override fun toString() = "InMemoryFileSystem"
+  fun allPaths(): MutableSet<File> {
```
Comment (swankjesse) [inline okhttp-testing-support/src/main/kotlin/okhttp3/internal/io/InMemoryFileSystem.kt:122]: MutableSet –> Set

File: okhttp-testing-support/src/main/kotlin/okhttp3/internal/io/InMemoryFileSystem.kt [hunk 2]
```diff
@@ -119,4 +119,7 @@ class InMemoryFileSystem : FileSystem, TestRule {
...
+    return files.keys
```
Comment (swankjesse) [inline okhttp-testing-support/src/main/kotlin/okhttp3/internal/io/InMemoryFileSystem.kt:123]: files.keys is a live view; `files.keys.toSet()` is a necessary defensive copy, if that’s what we want

**Review by swankjesse — COMMENTED** [review]
LGTM

**Pull request merged**